README.md

```diff
@@ -153,36 +152,41 @@ An error response object will have the following parameters.
 ```
 {
   error: true,
-  message: (may be undefined) String containing error message,
-  code: (may be undefined) String containing an error code,
-  response: (may be undefined) Object containing response from server,
+  message (may be undefined): String containing error message,
+  code (may be undefined): String containing an error code,
+  response (may be undefined): Object containing response sent from the server,
 }
 ```
 
 This library also exposes functions for encoding JavaScript values into Buffers which represent the bytes that a returned on-chain.
 
-- `Functions.encodeUint256(x)` takes a positive JavaScript integer number `x` and returns a 32 byte Buffer representing `x` as a `uint256` type in Solidity.
-- `Functions.encodeInt256(x)` takes a JavaScript integer number `x` and returns a 32 byte Buffer representing `x` as a `int256` type in Solidity.
-- `Functions.encodeString(x)` takes a JavaScript string `x` and returns a Buffer representing `x` as a `string` type in Solidity.
+- `Functions.encodeUint256` takes a positive JavaScript integer number and returns a Buffer of 32 bytes representing a `uint256` type in Solidity.
+- `Functions.encodeInt256` takes a JavaScript integer number and returns a Buffer of 32 bytes representing a `int256` type in Solidity.
+- `Functions.encodeString` takes a JavaScript string and returns a Buffer representing a `string` type in Solidity.
+
+Remember, it is not required to use these encoding functions.  The JavaScript code must only return a Buffer which represents the `bytes` array that is returned on-chain.
 
 ## Modifying Contracts
 
-Client contracts which initiate a request and receive a fulfillment can be modified for specific use cases. The only requirements are that the client contract extends the `FunctionsClient` contract and the `fulfillRequest` callback function never uses more than 300,000 gas.
+Client contracts which initiate a request and receive a fulfillment can be modified for specific use cases. The only requirements are that the contract sucessfully calls *sendRequest* in the *FunctionsOracle* contract and correctly implement their own *handleOracleFulfillment* function.  At this time, the maximum amount of gas that *handleOracleFulfillment* can use is 300,000. See *FunctionsClient.sol* for details.
```

Comment:
typo "sucessfully" 
"implement" -> "implements"

Reply:
Technically, the user doesn't have to implement anything for the *sendRequest* function.  They must successfully call sendRequest and correctly implement their own *handleOracleFulfillment*

Reply:
These two typos are still here.